package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "theme-dracula",
-	"version": "2.24.2",
+	"version": "2.24.3",
```

Comment:
Please revert this. The version number will get updated when a new version is ready for deployment.

Reply:
I have made changes requested, reverted version number